### PR TITLE
Add table for data 

### PR DESCRIPTION
**Pull request by ruben1**

### CODE REVIEW

File: src/client/app/modules/routes/edit-routes.controller.js [hunk 1]
```diff
@@ -22,6 +22,13 @@
       businessLogic: ''
     };
 
+    vm.sortType     = 'id'; // set the default sort type
+    vm.sortReverse  = false;  // set the default sort order
+    vm.searchData   = '';     // set the default search/filter term
+  
+
+    vm.user = vm.formInfo.data[0];
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/routes/edit-routes.controller.js]: what does this signify?

File: src/client/app/modules/routes/edit-routes.controller.js [hunk 2]
```diff
@@ -107,6 +114,7 @@
           vm.formInfo.data = res.data;
           vm.formInfo.methodKeys = Object.keys(vm.formInfo.methods);
           vm.formInfo.businessLogic = res.businessLogic;
+          vm.user = vm.formInfo.data[0];
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/routes/edit-routes.controller.js]: flag for above discussion

File: src/client/app/modules/routes/edit-routes.jade [hunk 3]
```diff
@@ -12,6 +12,28 @@ form(name="editForm")
     label Persistent data
     p {{ vm.formInfo.data.length === 0 ? 'No data' : vm.formInfo.data | json }}
 
+    .alert.alert-info
+    p Sort Type: {{ vm.sortType }}
+    p Sort Reverse: {{ vm.sortReverse }}
+    p Search Query: {{ vm.searchData }}
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/routes/edit-routes.jade]: should clear these out

File: src/client/app/modules/routes/edit-routes.jade [hunk 4]
```diff
@@ -12,6 +12,28 @@ form(name="editForm")
...
+    form
+      .form-group
+        .input-group
+          .input-group-addon
+            i.fa.fa-search
+          input.form-control(type='text', placeholder='Search', ng-model='vm.searchData')
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/routes/edit-routes.jade]: this style of cascading classes is not intuitive or needed if it'll only apply to one element. (also, didn't know it was possible -- really neat)

**Comment by AndrewSouthpaw** [comment]
reviewed.

File: src/client/app/modules/routes/edit-routes.controller.js (hunk 1 above)
Comment (ruben1) [inline src/client/app/modules/routes/edit-routes.controller.js]: they are default values sorting, reverse and keyword search

**Pull request merged**